tests/Rule/LatteTemplatesRule/PresenterWithoutModule/LatteTemplatesRuleForPresenterTest.php

```diff
@@ -143,6 +163,11 @@ public function testComponents(): void
     public function testFilters(): void
     {
         $this->analyse([__DIR__ . '/Fixtures/FiltersPresenter.php', __DIR__ . '/Fixtures/ParentPresenter.php'], [
+            [
+                'Trying to invoke mixed but it\'s not a callable.',
```

Comment:
this is covered by message Undefined latte filter, imho we can ignore it

Reply:
For example when user passes callable in variable and tries to invoke it in template then same error will occur. So we cannot ignore it. Only option would be conditional ignore I mwntioned - ignore this error only if undefined filter error is present on same location